src/main/java/dev/task/dndquest/validator/CharacterClassConstraintValidator.java

```diff
@@ -0,0 +1,23 @@
+package dev.task.dndquest.validator;
+
+import dev.task.dndquest.service.PlayCharacterClassService;
+import dev.task.dndquest.validator.annotation.CharacterClassConstraint;
+import jakarta.validation.ConstraintValidator;
+import jakarta.validation.ConstraintValidatorContext;
+import org.springframework.beans.factory.annotation.Autowired;
+
+public class CharacterClassConstraintValidator
+         implements ConstraintValidator<CharacterClassConstraint, String> {
+    @Autowired
```

Comment:
field injection is for poor people. We use constructor wiering

Reply:
ok